modules/battenberg/1.2/src/reference_correction.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/python
```

Comment:
Shouldn't this be 
#!/usr/bin/env python

Reply:
Right, it should be `#!/usr/bin/env python`, I changed it in the last commit before approving here:
https://github.com/LCR-BCCRC/lcr-modules/pull/171/commits/381887835f60446f39d83747fb009201c9d53cba